packages/orun/orun.0.0.1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "Run benchmarks and measure performance"
+maintainer: "Shakthi Kannan <author@shakthimaan.com>"
+authors: "Stephen Dolan <stephen.dolan@cl.cam.ac.uk>"
+license: "MIT"
+homepage: "https://github.com/ocaml-bench/orun"
+bug-reports: "https://github.com/ocaml-bench/orun/issues"
+dev-repo: "git+https://github.com/shakthimaan/orun"
+depends:
+  [ "ocaml"
+    "dune" {>= "1.2"}
+    "re"
+    "cmdliner"
+    "yojson"
+]
+depexts: [
+  ["libdw-dev"] {os-family = "debian" | os-family = "ubuntu"}
+  ["elfutils-devel"] {os-family = "centos" | os-family = "fedora" | os-family = "oraclelinux"}
+  ["libdw-devel"] {os-family = "opensuse"}
+]
+build: [
+  ["ocaml-update-c" "wait4.c"] {ocaml:update-c}
```

Comment:
this is the first time I see this `update-c` variable. It does not seem to be set by `ocaml-config`, where does it come from?

Reply:
The change came from this commit:
https://github.com/ocaml-bench/orun/commit/08ff09caff1b494212db5402257a73b457e2c9d2

The tool is available in https://github.com/ocaml-multicore/ocaml-multicore/blob/parallel_minor_gc/tools/ocaml-update-c.

Reply:
I'm not seeing this variable set in https://github.com/ocaml-multicore/multicore-opam/blob/master/packages/ocaml/ocaml.4.10.0/files/gen_ocaml_config.ml.in. Is that only set manually?

Reply:
From Stephen Dolan: "The tool was used when both Multicore and trunk had incompatible C APIs, so that script made some adjustments so that code written for trunk would work on multicore. It might no longer be necessary."